Description: Replace Kali Linux with Parrot GNU/Linux in the msfupdate error.
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2017-07-27

--- a/msfupdate
+++ b/msfupdate
@@ -135,7 +135,7 @@ class Msfupdate
 
     Dir.chdir(@msfbase_dir) do
       if apt?
-        stderr.puts "[-] ERROR: msfupdate is not supported on Kali Linux."
+        stderr.puts "[-] ERROR: msfupdate is not supported on Parrot GNU/Linux."
         stderr.puts "[-] Please run 'apt update; apt install metasploit-framework' instead."
       elsif binary_install?
         update_binary_install!
